test/lib/plugin.js

```diff
@@ -0,0 +1,8 @@
+exports.defineTags = function(dictionary) {
+    dictionary.defineTag("template", {
+        onTagged: function(doclet, tag) {
+            doclet.tags = doclet.tags || [];
+            doclet.tags.push(tag);
```

Comment:
I think the real fix here is to use the new output jsdoc produces for `@template` rather than hack it back into the "unknown tags" list.